sentry_sdk/utils.py

```diff
@@ -1558,16 +1558,16 @@ def match_regex_list(item, regex_list=None, substring_matching=False):
     return False
 
 
-def is_sentry_url(hub, url):
-    # type: (sentry_sdk.Hub, str) -> bool
+def is_sentry_url(client, url):
```

Comment:
We should also mention this in the migration guide probably.

Reply:
Yes. I will update the migration guide!